web/src/features/charts/bar-breakdown/utils.ts

```diff
@@ -78,7 +76,7 @@ export const getProductionData = (data: ZoneDetail): ProductionDataType[] =>
     const storage = data.storage?.[generationMode as ElectricityStorageKeyType];
 
     // Production CO₂ intensity
-    const gCo2eqPerkWh = getProductionCo2Intensity(mode, data);
+    const gCo2eqPerkWh = getProductionCo2Intensity(mode, data) || 0;
```

Comment:
This can potentially be null or undefined if we add the check here that sets it to 0 we avoid errors later.

Reply:
I made the function return 0 instead of handling it here :)